src/api/roles/client/client.py

```
@@ -599,38 +601,122 @@ def get_review(coach_id: int, db = Depends(get_session), acc: Account = Depends(
 
     if acc is None:
         raise HTTPException(404, detail="Account not found")
-    
-    coach_request = db.query(ClientCoachRequest).filter(ClientCoachRequest.client_id == acc.client_id).first()
 
-    if not coach_request.is_accepted:
-        raise HTTPException(403, detail="You are not authorized to see this coach until the request is accepted")
-    
+    coach_request = db.query(ClientCoachRequest).filter(
+        ClientCoachRequest.client_id == acc.client_id,
+        ClientCoachRequest.is_accepted == True
```

Ruff (0.15.12)

[error] Avoid equality comparisons to `True`; use `ClientCoachRequest.is_accepted:` for truth checks

Replace with `ClientCoachRequest.is_accepted`

(E712)